Resources/Prototypes/Entities/Clothing/OuterClothing/base_clothingouter.yml

```diff
@@ -146,6 +146,7 @@
       flatReductions:
         Heat: 10 # the average lightbulb only does around four damage!
     slots: OUTERCLOTHING
+  - type: BlockDefibrillator // Delta-V
```

Comment:
Remember to double check all the weird edge cases for hardsuits. There are a lot :(